README.md

```diff
@@ -1,13 +1,14 @@
 # Windows Console Documentation
-Welcome to the Windows Console documentation repo. The MSDN Console Docs are generated from the markdown stored in this repo.
-The published docs can be found at [https://docs.microsoft.com/windows/console/](https://docs.microsoft.com/windows/console/)
+
+Welcome to the Windows Console documentation repo. The published articles are generated from the markdown stored in this repo and published at [Windows Console documentation](https://docs.microsoft.com/windows/console/).
```

Comment:
"https://docs.microsoft.com/windows/console/" could be changed to a relative link.

Reply:
Files at the root of a repo are not published and require an absolute link when viewed in GitHub.